rosdep/python.yaml

```diff
@@ -4047,6 +4047,16 @@ python3-pep8:
 python3-pkg-resources:
   debian: [python3-pkg-resources]
   ubuntu: [python3-pkg-resources]
+python3-pyparsing:
+  arch: [python-pyparsing]
+  debian: [python3-pyparsing]
+  fedora: [python3-pyparsing]
+  gentoo: [dev-python/pyparsing]
+  ubuntu: [python3-pyparsing]
+python3-pytest:
+  arch: [python-pytest]
+  ubuntu:
+    bionic: [python3-pytest]
```

Comment:
I would suggest to just use `ubuntu: [python3-pytest]` in order to not need to maintain this in the future. The specific version requirement of `ament_tools` is just an arbitrary line in the sand. Any other package might have a different threshold.

Reply:
This is contrary to your first piece of feedback on this pull request.

Reply:
My first comment was in the context of your proposal to add the rosdep keys in order to be used for the ROS 2 packages. In that context I mentioned that this will not work since we require a newer version.

During this discussion we have established that the ROS 2 packages can't use this new rosdep key since there is no Debian package available which is new enough. And in the context of just adding these rosdep keys for other packages I think it should not be limited to bionic.